### DISCUSSION TITLE
Proof of Academic Status Not Accepted for Georgia Tech Online Graduate Program

### DISCUSSION THREAD

**Discussion by BigTMiami** [community/community#55851] [GitHub Education]
### Hi everyone,

I am getting a Masters in Computer Science from Georgia Tech in their online program (OMS CS).  I am based in Miami and the application logs that  my location is not in Atlanta and fails me.  I have provided every different set of info I can and it still doesn't accept me.  I would really like to participate in the students Github program.  Is there any person \ email that I can contact to escalate this?

**Comment by samus-aran** [comment]
Hi @BigTMiami 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
